### PR TITLE
Final pass on platformization

### PR DESCRIPTION
**Pull request by ravindk89**

### CODE REVIEW

**Comment by ravindk89** [comment]
@djwfyi can you pull this and just confirm that on a fast pass things look OK? these are largely non-material changes

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line, and also
 # from the environment for the first two.
-SPHINXOPTS    ?= -n -j "auto" -w "build.log"
+SPHINXOPTS    ?= -n -w "build.log"
```
Comment (harshavardhana) [inline Makefile:6]: we should keep this

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (ravindk89) [reply Makefile:6]: SO I get a warning from the sitemap builder with this flag enabled, *but* it doesn't seem to impact the actual contents of the file. Not sure - I am reaching out to the maintainer to understand what the concern is.

Comment (harshavardhana) [reply Makefile:6]: Without this it is not parallelized and slow.

**Pull request merged**